streams/shared/src/main/scala/zio/stream/ZTransducer.scala

```diff
@@ -731,6 +731,18 @@ object ZTransducer {
       }
     }
 
+  /**
+   * Decodes chunks of US-ASCII bytes into strings.
+   *
+   * This transducer uses the String constructor's behavior when handling malformed byte
+   * sequences.
+   */
+  val usASCIIDecode: ZTransducer[Any, Nothing, Byte, String] =
+    ZTransducer.fromPush {
+      case Some(chunk) => ZIO.succeedNow(Chunk.single(new String(chunk.toArray, StandardCharsets.US_ASCII)))
```

Comment:
Will this throw an exception when non-ASCII bytes are encountered? If so, we should use `ZIO(...)` rather than `succeedNow`.

Reply:
@iravid it doesn't seem to be the case. From the docs:
```
This method always replaces malformed-input and unmappable-character sequences with this charset's default replacement string. 
```
In `ASCII`'s case, it returns `���`. No exception thrown